<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactUsWindow</class>
 <widget class="QDialog" name="ContactUsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="contactUsTitle">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>13</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contact us!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dialog1Title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please leave us your feedback below, and we would get back to you:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mobileNumberTitle">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>159</y>
     <width>108</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobile Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>203</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Feedback:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mobileNumberDisplay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailDisplay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="feedbackDisplay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>361</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackStatusDisplay">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Feedback Status:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
